css/properties/line-break.json

```
@@ -118,24 +118,22 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "≤83"
+                "version_added": "25"
               },
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": "≤72"
-              },
-              "firefox_android": {
-                "version_added": "79"
+                "version_added": "69"
               },
+              "firefox_android": "mirror",
               "ie": {
                 "version_added": false
               },
               "oculus": "mirror",
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "≤13.1"
+                "version_added": "8"
```

In Safari 6.2 / 7 the default value changed from "normal" to "auto", so I think it makes sense that "auto" was supported from Safari 7.

```suggestion
                "version_added": "7"
```